tianshou/highlevel/experiment.py

```diff
@@ -114,6 +114,22 @@ class ExperimentConfig:
     policy_persistence_mode: PolicyPersistence.Mode = PolicyPersistence.Mode.POLICY
     """Controls the way in which the policy is persisted"""
 
+@dataclass
+class EvaluationProtocalExperimentConfig(ExperimentConfig):
+    train_seed_mechanism: Literal["consecutive"]|Literal["repeat"] = "consecutive" # or repeat, if all train seeds are supposed to be the same or consecutive numbers, compare seed function in venvs.py
```

Comment:
Check in `__post_init__` that intersection of train seed and test seeds is empty